clients/src/main/java/org/apache/kafka/common/metrics/Sensor.java

```diff
@@ -41,7 +41,34 @@
     private volatile long lastRecordTime;
     private final long inactiveSensorExpirationTimeMs;
 
-    Sensor(Metrics registry, String name, Sensor[] parents, MetricConfig config, Time time, long inactiveSensorExpirationTimeSeconds) {
+    public enum LogLevel {
+        SENSOR_INFO, SENSOR_DEBUG;
```

Comment:
INFO would map to the level we use for normal production runs, and DEBUG could be used to optimize the job in the development or instrumentation/debugging phase. Can't think of any more use cases, maybe TRACE could be a finer level, but personally have never found that useful.